scipy/fftpack/tests/test_czt.py

```diff
@@ -0,0 +1,70 @@
+# This program is public domain
```

Comment:
Indeed, shouldn't be in the docstring. And it is present also in a comment in ``czt.py``.